### PR TITLE
Add brainglobe-atlasapi and pymcubes dependency

### PR DESCRIPTION
**Pull request by willGraham01**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

(Will close https://github.com/pmneila/PyMCubes/issues/44)

### CODE REVIEW

**Comment by adamltyson** [comment]
Happy to be a maintainer :)

**Comment by alessandrofelder** [comment]
Happy to be a maintainer!

**Comment by pmneila** [comment]
Happy to be a maintainer.

File: recipes/brainglobe-atlasapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - treelib
+    - vedo
+    - vtk
+    - xmltodict
+
+test:
+  imports:
+    - brainglobe_atlasapi
+  commands:
+    - pip check  # [not win]
```
Comment (ocefpaf) [inline recipes/brainglobe-atlasapi/meta.yaml:48]: B/c this is noarch we don't necessarily require Windows to pass. However, I want to check the logs what is going on here to ensure the package will be functional on Windows.

```suggestion
    - pip check
```

**Comment by ocefpaf** [comment]
The Windows failure seems to be a non-zero exit but hard to understand what is going on by just looking at the logs. Probably OK as noarch. However, Linux is failing now :-/

**Comment by willGraham01** [comment]
@ocefpaf Thanks for taking a look - have fixed the linux build (was a runtime library issue).

File: recipes/pymcubes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
```
Comment (ocefpaf) [inline recipes/pymcubes/meta.yaml:22]: I don't believe you need those here. Only in build. Can you give it a try?

File: recipes/pymcubes/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - python
+    - setuptools
+    - wheel
+    - cython
+    - numpy
```
Comment (ocefpaf) [inline recipes/pymcubes/meta.yaml:27]: You probably need to repeat cython, numpy, and python on host.

Comment (willGraham01) [reply recipes/pymcubes/meta.yaml:25]: Did you mean on `run`? They're already specified as `host` dependencies.

Comment (ocefpaf) [reply recipes/pymcubes/meta.yaml:25]: Sorry I meant on build.

**Pull request merged**